src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        final String formatString = "dd.MM.yyyy";
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(formatString);
+        String [] parsedData;
+        LocalDate dateFromD;
+        LocalDate dateToD;
+        LocalDate day;
+        int employeeSalary;
+
+        try {
+            dateFromD = LocalDate.parse(dateFrom, formatter);
+            dateToD = LocalDate.parse(dateTo, formatter);
+        } catch (DateTimeParseException ex) {
+            System.out.println("Not correct date!");
+            return null;
```

Comment:
```suggestion
            throw new RuntimeException("Could not parse date ", e)
```

It's better to throw some exception here